src/state/auth/selectors.js

```diff
@@ -14,3 +14,7 @@ export function isSessionExpired(state) {
   const expiresAt = state.session.get('expiresAt', 0);
   return new Date().getTime() >= expiresAt;
 }
+
+export function getAuthStarted(state, defaultsTo = false) {
+  return state.auth.session.get('authStarted', defaultsTo);
```

Comment:
What is this `.session` thing? I just don't get where it gets set. Is that a Rehan thing or a Redux thing?

Reply:
I believe it comes from https://github.com/mozilla/delivery-console/blob/53757a8ad880df61121fbf06afba555203ad47cf/src/state/auth/reducers.js#L37 so it's rather a `redux.combineReducer` thing ;)

Reply:
update to `'inProgress'`